### ISSUE TITLE
Obsolete documentation file

### ISSUE THREAD

**Issue by albert-github** [CGAL/cgal#3753]
During the post processing phase of `make doc_with_postprocessing` the file `deprecated.html` is copied to the `Manual` output directory.
- as far as I can see the is never referenced
- the file refers to CGAL 4.3, independent of the used version
- the file is not XHTML compliant

The original file can be found, for the different versions, at:
```
./doc/resources/1.8.13/deprecated.html
./doc/resources/1.8.14/deprecated.html
./doc/resources/1.8.4/deprecated.html

```

the copying is done nearly at the end of the file `Documentation/doc/scripts/html_output_post_processing.py` in the lines:
```
    #copy deprecated.html
    shutil.copy(path.join(resources_absdir,"deprecated.html"),path.join("Manual/", "deprecated.html"))
```

As I'm not 100%sure about the usage with doxygen version 1.8.4 I have not created a pull request to remove the files and code, but when the support for 1.8.4 is removed I think these files / code should also be removed.

**Comment by sloriot** [comment]
This file was used by our rewrite rules (@lrineau  is it still the case? I no longer find it on the server). Actually, this has been introduced when we switched from our internal doc tool to doxygen to indicate cases where the documentation was missing (IIRC).

**Comment by lrineau** [comment]
[**@sloriot**](https://github.com/sloriot) commented on [Mar 14, 2019, 5:03 PM GMT+1](https://github.com/CGAL/cgal/issues/3753#issuecomment-472936140 "2019-03-14T16:03:13Z - Replied by Github Reply Comments"):
> This file was used by our rewrite rules (@lrineau  is it still the case? I no longer find it on the server). Actually, this has been introduced when we switched from our internal doc tool to doxygen to indicate cases where the documentation was missing (IIRC).

That is still the case. The rewrite path is quite complicated. See https://github.com/CGAL/cgal-web/blob/3b8835eb67d14454c36b1abfca953e6ecf01e2d1/Manual/latest/doc_html/.htaccess#L17 (private repo, sorry Albert).

Then on cgal.geometryfactory.com, there is an Apache rewrite map in use, `www.cgal.org/rw_rules/rewrite_pages.txt`, that redirects to `Manual/deprecated.html` if needed.

Live example:
https://www.cgal.org/Manual/latest/doc_html/cgal_manual/NumberTypeSupport_ref/Class_Root_of_2.html

One can see the original file here:
https://doc.cgal.org/Manual/latest/doc_html/cgal_manual/NumberTypeSupport_ref/Class_Root_of_2.html

**Comment by lrineau** [comment]
[**@lrineau**](https://github.com/lrineau) commented on [Mar 14, 2019, 5:16 PM GMT+1](https://github.com/CGAL/cgal/issues/3753#issuecomment-472942908 "2019-03-14T16:16:43Z - Replied by Github Reply Comments"):
> Live example:
> https://www.cgal.org/Manual/latest/doc_html/cgal_manual/NumberTypeSupport_ref/Class_Root_of_2.html

There is actually now three redirections:
```shellsession
$ curl -IL https://www.cgal.org/Manual/latest/doc_html/cgal_manual/NumberTypeSupport_ref/Class_Root_of_2.html | grep -E 'HTTP|Location'
HTTP/1.1 303 See Other
Location: https://cgal.geometryfactory.com/CGAL/Manual/latest/cgal_manual/NumberTypeSupport_ref/Class_Root_of_2.html
HTTP/1.1 302 Found
Location: http://doc.cgal.org/latest/Manual/deprecated.html
HTTP/1.1 302 Found
Location: https://doc.cgal.org/latest/Manual/deprecated.html
HTTP/1.1 200 OK
```

**Comment by albert-github** [comment]
No problem that I cannot access the full process.
As the file is still in use I think it would be good to adjust it to signal the right version of CGAL and make it (X)HTML compatible.

**Comment by lrineau** [comment]
Could this file be generated by Doxygen, instead of being an hard-coded file?
```patch
diff --git a/Documentation/doc/Documentation/main.txt b/Documentation/doc/Documentation/main.txt
index 22a8407f82e..055ab9ff51a 100644
--- a/Documentation/doc/Documentation/main.txt
+++ b/Documentation/doc/Documentation/main.txt
@@ -60,4 +60,12 @@ For releases X.Y, with 3.1 <= X.Y <= 4.1 visit
 </div>
 \endhtmlonly
 
+\page deprecated Removed documentation pages
+
+The documentation page you are trying to reach has been removed from
+the documentation since %CGAL 4.3 because it was deprecated.
+
+You can find <a href="https://doc.cgal.org/4.2">here the documentation
+of CGAL-4.2</a>, with the old documentation pages.
+
 */
```

Issue: the page is then visible in the navigation tree. Is there a way to remove it from the navtree?

**Comment by sloriot** [comment]
Why is the static version not fine?

**Comment by lrineau** [comment]
The problem is that it is static, and not updated:
- its HTML title is
  > CGAL 4.3 - Manual: $title
- its top bar is the one of the documentation of CGAL-4.3:
- its title area say "CGAL 4.3"
- and the HTML is the old one of CGAL-4.3, not modernized.

With time it might even get incompatible with our CSS rules.

**Comment by albert-github** [comment]
Is theer in the current documentation a place that has this redirecting?

**Comment by sloriot** [comment]
Fixed in #3771
